far2l/FARString.hpp

```diff
@@ -61,10 +61,10 @@ class FARString
 	{
 		static Content sEmptyData; //для оптимизации создания пустых FARString
 
-		volatile unsigned int m_nRefCount;
-		unsigned int m_nCapacity;	// not including final NULL char
-		unsigned int m_nLength;		// not including final NULL char
-		wchar_t m_Data[1];
+		volatile unsigned int m_nRefCount{0};
```

Comment:
please remove any initializers here, Content must be POD-like structure, see comment above